core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java

```diff
@@ -60,7 +60,9 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import static org.commonjava.indy.IndyContentConstants.CHECK_CACHE_ONLY;
```

Comment:
If we moved IndyContentConstants to model-java, then we could use this constant in the client api too, right?

Probably not a big deal, but it might be nice to have a single place to change things like this.

Reply:
Which model-java? 
The IndyContentConstants is now in /models/core-java. IndyClientHttp can use it.